### ISSUE TITLE
Using no flickering causes issues

### ISSUE THREAD

**Issue by kodenamekrak** [bsq-ports/Chroma#35]
If no flickering is selected on lighting options and custom color events is enabled, chroma maps will take longer to load which is usually <10 seconds but in some cases 30+ seconds. Once in song it will stutter breifly a few times at the start of the song, and in [this map](https://beatsaver.com/maps/19901) specifically the game will completely freeze at 59 seconds. This problem does not occur with all effects or no effects

**Comment by Dr01000111** [comment]
if i can add on i think a fix would be to either add a warning or re route the song to use all effects and give a warning in game about no flickering either on start up or before you start a song

**Comment by LaidBackAries** [comment]
No flickering causes issues with colors being displayed properly.  I've been trying to fix my map's lighting all day. Boost colors were not showing up so I changed everything to 100% Chroma and it still wasn't working.  I just noticed that no flickering was on.  It's supposed to reduce the strobing effect (which I don't really have in my map) but instead it reduces the number of colors displayed.  :/
